.circleci/cimodel/data/simple/ios_definitions.py

```diff
@@ -45,6 +45,9 @@ def gen_tree(self):
             "build_environment": self.gen_job_name(),
             "ios_arch": self.arch_variant.name,
             "ios_platform": platform_name,
+            "when": {
```

Comment:
I dig this out of https://circleci.com/docs/configuration-reference/#logic-statements, it might not work as I think it should.  Let's see